src/main/java/no/fint/portal/nam/ObjectFactory.java

```diff
@@ -0,0 +1,273 @@
+package no.fint.portal.nam;
+
+import no.fint.portal.nam.model.*;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static no.fint.portal.nam.Constants.*;
+
+public final class ObjectFactory {
```

Comment:
Is this generated code?  If it is, you should either:

1. Generate the code from the source schema as part of the build
1. Add a comment indicating how it was generated, and how to re-generate the code from the schema.

Reply:
It's not generated code. The question is, is this a good way to do it?